GlucoseOnFhir/DeviceViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  DeviceViewController.swift
+//  GlucoseOnFhir
+//
+//  Created by Kevin Tallevi on 1/16/17.
+//  Copyright © 2017 eHealth Innovation. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SMART
+
+class DeviceViewController: UITableViewController {
+    public var device: Device!
+    let cellIdentifier = "DeviceCellIdentifier"
+    let sectionHeaderHeight: CGFloat = 75
+    
+    enum Section : Int {
+        case section0, section1, section2, section3, count
```

Comment:
Wouldn't it be clearer to use names? like `case identifier, type, manufacturer, model`?